src/js/utils/list-utils.jsx

```diff
@@ -6,26 +6,32 @@ export const hasMinimumRequiredRole = (role, highestUserRole) => {
 };
 
 
-export const findActions = (actionList, row, supportedActivities, highestUserRole) => {
-  // Firstly filter out by status if any provided
+export const findActions = (actionList, row, props) => {
+  const { supportedActivities = [], highestRole = [], customFilter } = props;
```

Comment:
I have made `supportedActivities`, `highestRole` and `customFilter` to be properties because we sometimes don't need to pass all of them.
So instead of passing `findActions(actions, row,  [], [], (action) => action.hasProperty)` 
we can do `findActions(actions, row,  { customFilter: (action) => action.hasProperty })`